scripts/build_and_install.sh

```
@@ -22,19 +22,25 @@ function detectCores() {
-        exit $?
+    # Check if this is a Windows mount point or native WSL Linux
+    if [[ "$(pwd)" == /mnt/* ]]; then
+        # Windows mount point - switch to Git Bash for Windows builds
+        echo "You're using WSL with Windows mount point. Switching to Git Bash for Windows build!" >&2
+        GIT_BASH_PATH_WIN=$(/mnt/c/Windows/system32/cmd.exe /C "where bash.exe" | grep -i Git | head -n 1 | tr -d '\n\r')
+        GIT_BASH_PATH_WSL=$(wslpath -u "$GIT_BASH_PATH_WIN")
+        echo "== GIT_BASH_PATH_WIN=$GIT_BASH_PATH_WIN"
+        echo "== GIT_BASH_PATH_WSL=$GIT_BASH_PATH_WSL"
+        if [[ -f "$GIT_BASH_PATH_WSL" ]]; then
+            THIS_BASE_NAME=$(basename "$0")
+            "$GIT_BASH_PATH_WSL" "$THIS_BASE_NAME" $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)